### PR TITLE
Use ES6 classes to define ChartListContainer

### PR DESCRIPTION
**Pull request by openjck**
Open to discussion. Personally I find the object-oriented approach easier to read.

### CODE REVIEW

File: distributionviewer/core/static/js/app/components/containers/chart-list-container.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-// numCharts is updated and provided to ChartList just for demonstration
-// purposes, and so that ChartList doesn't just have to include <ExampleChart />
-// 30 times in its source.
-const ChartListContainer = React.createClass({
-  getInitialState: function() {
-    return {
-      numCharts: 0
-    }
-  },
+export class ChartListContainer extends React.Component {
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/containers/chart-list-container.js:4]: This seems a little more verbose but it does make certain things like the constructor a bit more clear. Consistency will be important I guess.

**Comment by spasovski** [comment]
r+ after rebase

File: distributionviewer/core/static/js/app/components/containers/chart-list-container.js (hunk 1 above)
Comment (openjck) [inline distributionviewer/core/static/js/app/components/containers/chart-list-container.js:4]: Yeah. This particular line is definitely more verbose, but at the same time it allows us to forego the commas and colons and other syntactic fluff that passing objects entails.

**Comment by openjck** [comment]
Updated! :smile:

**Pull request merged**